modules/rating/src/main/Glicko.scala

```diff
@@ -33,7 +35,10 @@ case class Glicko(
       volatility > 0 &&
       volatility < (Glicko.maxVolatility * 2)
 
-  def cap = copy(volatility = volatility min Glicko.maxVolatility)
+  // for rating refunds, aging should somehow increase the rating deviation (hence abs)
```

Comment:
Age is based on what I read in the Glicko system description; however the Java implementation already does this if properly consumed (which it is not!!):
http://glicko.net/glicko/glicko.pdf